### PR TITLE
Cut right dropdown margin if no hyperlink, image, submenu, and add dropdown arrow visible option.

### PR DESCRIPTION
**Pull request by roachsinai**
1. if no hyperlink, image, submenus, no margin on the right of dropdown.
2. could choose dropdown arrow on status bar always visible, never visible or visible as needed(mouse click).

image

### CODE REVIEW

File: plasmoid/contents/ui/config/ConfigAppearance.qml [hunk 1]
```diff
@@ -7,49 +7,71 @@ import org.kde.plasma.extras 2.0 as PlasmaExtras
...
     id: page
-    
+
     property alias cfg_width: width.value
     property alias cfg_compactLabelMaxWidth: compactLabelMaxWidth.value
     property alias cfg_height: height.value
-    property alias cfg_dropdownvisible: dropdownvisible.checked
-    
-    
+    property alias cfg_d_arrownevervisible: d_arrownevervisible.checked
```
Comment (shocklateboy92) [inline plasmoid/contents/ui/config/ConfigAppearance.qml]: Camel case these please (e.g `arrownevervisible` -> `arrowNeverVisible`).

**Review by shocklateboy92 — APPROVED** [review]
Looks good. Minor point about variable names. 
Would appreciate if you can update them before merging.

File: plasmoid/contents/ui/config/ConfigAppearance.qml (hunk 1 above)
Comment (roachsinai) [reply plasmoid/contents/ui/config/ConfigAppearance.qml]: You mean `cfg_d_arrownevervisible: d_arrownevervisible` to `cfg_d_arrowNeverVisible: d_arrowNeverVisible`?

Or only `d_arrownevervisible` to `d_arrowNeverVisible`?

Comment (roachsinai) [reply plasmoid/contents/ui/config/ConfigAppearance.qml:14]: You mean `cfg_d_arrownevervisible: d_arrownevervisible` to `cfg_d_arrowNeverVisible: d_arrowNeverVisible`?

Or only `d_arrownevervisible` to `d_arrowNeverVisible`?

Comment (shocklateboy92) [reply plasmoid/contents/ui/config/ConfigAppearance.qml]: The former, so the variable and the id both follow camel case. (Also applies to the two lines below) 
Thanks.

Comment (roachsinai) [reply plasmoid/contents/ui/config/ConfigAppearance.qml]: @shocklateboy92 review is needed!

**Pull request merged**